hudi-common/src/main/java/org/apache/hudi/common/util/HoodieCommonKryoRegistrar.java

```diff
@@ -80,7 +79,6 @@ public void registerClasses(Kryo kryo) {
         //kryo.register(BootstrapRecordPayload.class);
         AWSDmsAvroPayload.class,
         HoodieAvroPayload.class,
-        HoodieJsonPayload.class,
```

Comment:
We should not be removing any payload class from Kryo registrar. When records are shuffled then serialization takes place based on what classes have been registered here. In case there are records with HoodieJsonPayload that are shuffled, then it could result in "task not serializable" error. Let's keep the payload in hudi-common.

Reply:
Fixed it.